### PR TITLE
Create profanity-filter.yml

### LINKED ISSUE #403: Add profanity filter workflow

### PR DESCRIPTION
**Pull request by IEvangelist**
Adding profanity filter GitHub Action workflow.

Fixes #403

### CODE REVIEW

File: .github/workflows/profanity-filter.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - name: Print actor
+      run: |
+        echo ${{ github.actor }}
+
```
Comment (IEvangelist) [inline .github/workflows/profanity-filter.yml:23]:
```suggestion
```

File: .github/workflows/profanity-filter.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: Profanity filter
+
+on:
+  issue_comment:
```
Comment (BillWagner) [inline .github/workflows/profanity-filter.yml:4]: Does `issue_comment` include PR comment / review comment? If not, this is worth adding.

**Review by BillWagner — COMMENTED** [review]
This LGTM. I had one question for completeness, and then :shipit:

File: .github/workflows/profanity-filter.yml (hunk 2 above)
Comment (IEvangelist) [reply .github/workflows/profanity-filter.yml:4]: Review comments have not been considered, as we're usually not the one's using profanity. But that might be worth adding support for. And for `issue_comment`, that covers `pull_request_comment`, see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_comment-use-issue_comment.

**Review by BillWagner — APPROVED** [review]

**Pull request merged**